recipes/acl/all/conanfile.py

```diff
@@ -71,4 +71,10 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("pkg_config_name", "libacl")
+        self.cpp_info.set_property("cmake_file_name", "acl")
+        self.cpp_info.set_property("cmake_target_name", "acl::acl")
+        self.cpp_info.set_property("cmake_target_aliases", ["acl"])
         self.cpp_info.libs = ["acl"]
+        # TODO: to remove in conan v2
+        self.cpp_info.names["cmake_find_package"] = "acl"
+        self.cpp_info.names["cmake_find_package_multi"] = "acl"
```

Comment:
```suggestion
```
I would skip the Conan v1 part, since this has no effect anyway (the cmake_target_name and cmake_file_name at least have some informational value for the consumers).